src/helpers/isValidMAccount.ts

```diff
@@ -0,0 +1,2 @@
+export const isValidMAccount = (value: string) =>
+  value.startsWith("M") && value.length === 69;
```

Comment:
Why not use the SDK's validation methods? e.g. `try { decodeAddressToMuxedAccount(value, true) }`

Reply:
We use `StrKey.isValidEd25519PublicKey` to validate G-address, and `StrKey` didn't show any options for M-address in VS Code. I have to admit that I didn't look deeper into that at the time. 😊

It looks like we have [`isValidMed25519PublicKey`](https://stellar.github.io/js-stellar-base/StrKey.html#.isValidMed25519PublicKey) on `StrKey`, but it's missing in [types](https://github.com/stellar/js-stellar-base/blob/master/types/index.d.ts#L738). Should we use that to validate M-addresses to stay consistent once it's available? @Shaptic

Reply:
Good point, hard to expect you to use a validation method I never exported for use! :wink: I'll make an issue to track it for myself, and I'll let you know once that gets exposed.